data-prepper-plugins/s3-source/src/test/java/com/amazon/dataprepper/plugins/source/SqsWorkerTest.java

```diff
@@ -72,6 +77,8 @@ void processSqsMessages_should_return_number_of_messages_processed() {
         final int messagesProcessed = sqsWorker.processSqsMessages();
 
         assertThat(messagesProcessed, equalTo(1));
+        verify(s3Service, times(1)).addS3Object(any(S3ObjectReference.class));
+        verify(sqsClient, times(1)).deleteMessageBatch(any(DeleteMessageBatchRequest.class));
```

Comment:
These tests should use `ArgumentCaptor<DeleteMessageBatchRequest>` to capture the actual argument and then inspect it to be sure it has the correct values.